chsdi/models/vector/uvek.py

```diff
@@ -310,27 +322,34 @@ class unf_pers_casualties(Base, Vector):
 
 
 class unf_pers_fuss(Base, Vector):
```

Comment:
While you're at it you could factorize that as described in https://github.com/geoadmin/mf-chsdi3/pull/1991